### ISSUE TITLE
About ConcatTetraPermuter

### ISSUE THREAD

**Issue by kimnupjuk** [PattanaikL/chiral_gnn#2]
Hello! thanks for providing your whole code.

As a baseline of my research I tried to reproduce your result with RS recognition task.

However, concattetrapermuter gave me an error.

I think         
nei_messages = torch.sum(nei_messages, dim=-2)
should be added between line 78 and 79 of tetra.py

If my thinking is wrong, would you please to tell me my mistake kindly?

**Comment by PattanaikL** [comment]
Yes, you're correct! I made a change to speed up the permutations, but forgot to sum across neighbors for ConcatTetraPermuter. Just fixed it
